Cargo.toml

```diff
@@ -109,5 +109,7 @@ rustc-std-workspace-core = { path = 'library/rustc-std-workspace-core' }
 rustc-std-workspace-alloc = { path = 'library/rustc-std-workspace-alloc' }
 rustc-std-workspace-std = { path = 'library/rustc-std-workspace-std' }
 
+libc = { path = "../libc" }
```

Comment:
Changes to the libc crate are also needed in order to add Trusty support, but the PR to do so hasn't yet been merged (https://github.com/rust-lang/libc/pull/2987). How should I handle this?

Reply:
Wait until a new libc version is published with your change and then update the version of libc with this PR.

In the meantime you can either do nothing or make libc refer to your git branch.

Reply:
I've opted to instead remove the standard library support from this PR. I'll start off by adding the new target definitions in this PR, then open another one to add libstd support once libc has been updated as needed.